packages/experimental/Button/src/ButtonColorTokens.windows.ts

```diff
@@ -27,9 +27,10 @@ export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t:
       iconColor: t.colors.neutralForeground1Pressed,
     },
     focused: {
+      // Focused border matches background so latency betwen native and JS visuals isn't as noticable
```

Comment:
Is it noticeable?

Reply:
Yeah, but I'm also looking for it and running the dev bundle.